README.md

```diff
@@ -49,21 +49,21 @@ docker run -p 8080:8080 -p 3000:3000 openfga/openfga run
 
 1. First, either clone this repo or curl the `docker-compose.yaml` file with the following command:
 
-   ```bash
-   curl -LO https://openfga.dev/docker-compose.yaml
-   ```
+```shell
```

Comment:
Before this was indented so that it falls under the (1), now it appears to be it's own section. Was this intentional?

Reply:
good point! let me review that